src/scripts/main.js

```diff
@@ -1 +1,16 @@
 'use strict';
+
+const nav = document.querySelector('.gallery__list');
+const imgLarge = document.querySelector('.gallery__large-img');
+
+nav.addEventListener('click', (events) => {
+  events.preventDefault();
```

Comment:
Calling `event.preventDefault()` unconditionally might not be appropriate. Consider checking if the event target is indeed a link that should prevent default behavior.